README.md

```diff
@@ -6,17 +6,19 @@ This repository contains the source files for the kubescape.io website.
 
 ### Local testing
 
-We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. 
+We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme to built the website. 
```

Comment:
This is the readme for the website repo. The next line talks about building and testing, so I don't think we need to explain what these are for 🙂